### ISSUE TITLE
Valid excel range reference is considered invalid by EPPlus (ExcelAddress.IsValidAddress function)

### ISSUE THREAD

**Issue by lafar6502** [EPPlusSoftware/EPPlus#1101]
### EPPlus usage

Commercial use (I have a commercial license)

### Environment

Windows/.Net framework 4.8

### Epplus version

6.2.9

### Spreadsheet application

Excel

### Description

Using EPPlus for performing calculations based onExcel from a C# application (so we can avoid using Excel server side).

And we're using named cell ranges for referring to particular cell or region in the spreadsheet.
Previously we've been using EPPlus v 4.x.x and i think it worked correctly (however, i might be mistaken)

but after upgrading to 6.2.9 (and 6.2.10 too) EPPlus doesnt consider this reference to be valid excel address:

```
address = "Sheet1!Table313[#All]"
v = OfficeOpenXml.ExcelAddress.IsValidAddress(address)
//v == false
```
the address Sheet1!Table313[#All] refers to an Excel Table together with its headers
this is name manager in Excel, showing the problematic reference
![image](https://github.com/EPPlusSoftware/EPPlus/assets/1706814/1d9a60c1-b8a6-4a59-8736-8190651bda80)

**Comment by OssianEPPlus** [comment]
Thanks for letting us know! We're looking into this

**Comment by JanKallman** [comment]
This is bug. We will provide a fix for this shortly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.2.11 and 7.0.0

**Comment by lafar6502** [comment]
thanks
